lib/scala/runtime-version-manager/src/main/scala/org/enso/runtimeversionmanager/runner/Runner.scala

```diff
@@ -224,18 +224,28 @@ class Runner(
       case None =>
         runtimeVersionManager.withEngineAndRuntime(engineVersion) {
           (engine, runtime) =>
-            NativeExecCommand.apply(
-              engineVersion.toString,
-              engine,
-              logger
-            ) match {
-              case Some(cmd) =>
-                prepareAndRunCommand(engine, cmd)
-              case None =>
-                prepareAndRunCommand(
-                  engine,
-                  JavaExecCommand.forRuntime(runtime)
-                )
+            val ensoLauncher      = Option(System.getenv(Runner.LAUNCHER_ENV_NAME))
+            val requiresJVMRunner = ensoLauncher.exists(_.equals("shell"))
```

Comment:
The name `shell` is technically misleading. There is no _shell_ anywhere 
- in fact `project-manager` bypasses the launcher completely
- invokes directly `java`command with arguments referring to various JARs inside of engine
- that needlessly enlarges the API between `project-manager` and engine
- ideally `project-manager` should only add `--jvm` and continue to invoke the `enso` launcher